### PR TITLE
SwipeLayout disabled after long press 

### LINKED ISSUE #1019: Swipe to refresh hides the list of files selected to delete
**Describe the bug**
In Device tab in Library screen, when you select some files to delete and swipe down to refresh it, the files selected to delete are no more selected, but the option to delete is still there, i.e. you can still delete those on pressing the delete button on the action bar.

**Expected behavior**
Upon selecting files to delete, swipe to refresh feature should be disabled.

**Steps to reproduce the behavior:**
1. Go to the device tab in the library screen.
2. Select files to delete.
3. Swipe down to refresh.
4. The selected files are no more selected but the user can still delete them.

**Screenshots**

swipe_before_delete

**Environment**
- Version of Kiwix Android : 2.4 Build 55
- Device : Redmi 4
- OS version : Nougat 7.1

### PR DESCRIPTION
**Pull request by sonusourav**
Fixes #1019 

Changes: 
1. When files are selected on long press, swipeRefreshLayout is disabled.
2. It is enabled back when CAB disappears.

Screenshots/GIF for the change: 

swipe_before_delete

### CODE REVIEW

**Comment by sonusourav** [comment]
@mhutti1  When I was working on this issue, I found another issue that selected files get unselected on screen rotation. Since this was not part of issue #1019 , I opened another issue #1023  for it.

**Comment by sonusourav** [comment]
@mhutti1  Please review it once.

**Comment by soloturn** [comment]
@sonusourav thanks a lot!

File: app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.java [hunk 1]
```diff
@@ -138,10 +138,16 @@ public void onItemCheckedStateChanged(ActionMode mode, int position, long id,
 
         if (checked) { // If the item was selected
           selectedViewPosition.add(position);
+          swipeRefreshLayout.setEnabled(
+              false);  //disabled to stop selected items getting deselected (issue #1019)
```
Comment (rashiq) [inline app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.java]: Can you please remove the line break here?

**Comment by sonusourav** [comment]
@soloturn could you please review.

**Comment by kelson42** [comment]
@sonusourav Please rebase your branch

**Comment by sonusourav** [comment]
@kelson42  I have rebased it. Could you please review now?

**Pull request merged**